zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -0,0 +1,152 @@
+zipkin:
```

Comment:
@anuraaga thanks for the suggestion! I will give spring boot profiles approach a try.